velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -83,6 +83,11 @@ class ArrowDataBufferSink : public ::arrow::io::OutputStream {
     return sink_->isClosed();
   }
 
+  void abort() {
+    sink_.reset();
```

Comment:
I saw we do `close` for Dwrf's Writer: https://github.com/facebookincubator/velox/blob/fed29ea2acce2145a01089a889943e1b3c5058e9/velox/dwio/dwrf/writer/WriterBase.h#L44-L46

Should we do the same?

Reply:
`sink_.reset()` would trigger `~LocalFileSink()` and call `Closable::destroy`, which would call `close()`.